pom.xml

```diff
@@ -189,6 +189,12 @@
             <artifactId>jackson-databind</artifactId>
             <version>${jackson.databind.version}</version>
         </dependency>
+        <!-- Mail sender -->
+        <dependency>
+            <groupId>javax.mail</groupId>
+            <artifactId>mail</artifactId>
+            <version>1.4.7</version>
```

Comment:
Version should be took out to ${javax.mail.varsion}

Reply:
@Greengrer  Strange thing to hear that from you, when you also push json-simple dependency without version property ;)

Reply:
That's because I'm a moron, but I'm trying to learn better.